src/qt/walletmodel.cpp

```diff
@@ -231,7 +231,7 @@ WalletModel::SendCoinsReturn WalletModel::prepareTransaction(WalletModelTransact
 
         // Reject absurdly high fee. (This can never happen because the
         // wallet never creates transactions with fee greater than
-        // m_default_max_tx_fee. This merely a belt-and-suspenders check).
```

Comment:
in "[refactor]: rename m_default_max_tx_fee to m_max_tx_fee" (https://github.com/bitcoin/bitcoin/pull/29278/commits/f1ff2b8f40edffc2b61240a976cff9a9fd51a110):

nit: this commit could be a scripted-diff

Reply:
Thanks, changed to a scripted-diff